Libplanet.Net.Tests/SwarmTest.Fixtures.cs

```diff
@@ -117,6 +117,18 @@ public partial class SwarmTest
                 host = IPAddress.Loopback.ToString();
             }
 
+            options ??= new SwarmOptions();
+            string type = Environment.GetEnvironmentVariable("TRANSPORT_TYPE");
+            switch (type)
+            {
+                case "tcp":
+                    options.Type = SwarmOptions.TransportType.TcpTransport;
+                    break;
+                case "netmq":
+                    options.Type = SwarmOptions.TransportType.NetMQTransport;
+                    break;
+            }
```

Comment:
What if `TRANSPORT_TYPE` is neither `tcp` nor `netmq`?

Reply:
It will use default option on `SwarmOptions` class (which is `TcpTransport` for now).